tests/spread/integration/bind9-dnsutils/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: Integration tests for bind9-dnsutils
+
+execute: |
+  rootfs="$(install-slices bind9-dnsutils_bins)"
+
+  chroot "${rootfs}/" delv -v
+  chroot "${rootfs}/" dig -v
+  chroot "${rootfs}/" mdig -v
+  chroot "${rootfs}/" nslookup -version
+  chroot "${rootfs}/" nsupdate -V
```

Comment:
FYI, since Canonical has been asking for more functional integration tests, it might be good to add some functional tests here as well.

For example, it might be easy to build some function tests for dig, nslookup & host, since you might be able to run them using `127.0.0.1`.
I.e.:

```shell
$ dig +short -x 127.0.0.1
localhost.
```
```shell
$ nslookup 127.0.0.1
1.0.0.127.in-addr.arpa	name = localhost.

Authoritative answers can be found from:

```

```shell
$ host 127.0.0.1
1.0.0.127.in-addr.arpa domain name pointer localhost.
```